vector/src/main/java/im/vector/app/core/extensions/TextInputLayout.kt

```diff
@@ -78,10 +78,15 @@ fun TextInputLayout.setOnImeDoneListener(action: () -> Unit) {
     }
 }
 
-fun TextInputLayout.setOnFocusLostListener(action: () -> Unit) {
+/**
+ * Set a listener for when the input has lost focus, such as moving to the another input field.
+ * The listener is only called when the view is in a resumed state to avoid triggers when exiting a screen.
+ */
+fun TextInputLayout.setOnFocusLostListener(lifecycleOwner: LifecycleOwner, action: () -> Unit) {
+    lifecycleOwner.lifecycle
```

Comment:
not sure this line is necessary?

Reply:
https://github.com/vector-im/element-android/pull/6741/commits/b375dd14f781c076482c87fa81ebaa54d70db78c